coding-the-legislation/37_inferences.md

```diff
@@ -0,0 +1,26 @@
+# Inferences
```

Comment:
I'm not sure this belong here.

The first pages (with numbers `10->50`) of the chapter "Coding the legislation" are very didactic, and introduce you progressively to the different concepts used in OpenFisca. It's full of examples, and really detailed.

This page is not in the tone of the tutorial, and probably doesn't have the same audience.

I'm ok with keeping this page in the  "Coding the legislation" chapter, as I don't see any better place, but I'd prefer not putting it right in the middle of the step by step tutorial.